### PR TITLE
Created extension card for Helm Plugin for Meshery Snapshot

### PR DESCRIPTION
**Pull request by Aviral0702**
**Description**

This PR creates a card for [Helm Plugin for Meshery Snapshots](https://github.com/meshery/meshery/issues/11866).

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@Aviral0702 Thank you for your contribution @Mandeep56Singh
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

File: collections/_extensions/helm-plugin-for-meshery-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+kind: Plugin
+userName: Layer5, Inc.
+userAvatarURL: https://png.pngtree.com/png-vector/20191104/ourmid/pngtree-businessman-avatar-cartoon-style-png-image_1953664.jpg
+type: Collaboration
+compatibility: 
+  - aws
+  - kubernetes
+extensionId: 2d54f372-10e5-4c4e-8d23-b9c35c68ce98
+logo: /assets/images/meshery-logo.svg
+image: /assets/images/Helm.svg
```
Comment (vishalvivekm) [inline collections/_extensions/helm-plugin-for-meshery-snapshot.md:13]:
```suggestion
image: /assets/images/helm-dark.svg
```

File: collections/_extensions/helm-plugin-for-meshery-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: item
+name: Helm Plugin for Meshery Snapshot
```
Comment (fitzergerald) [inline collections/_extensions/helm-plugin-for-meshery-snapshot.md:3]:
```suggestion
name: Helm Plugin for Kanvas Snapshot 
```

File: collections/_extensions/helm-plugin-for-meshery-snapshot.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - kubernetes
+extensionId: 2d54f372-10e5-4c4e-8d23-b9c35c68ce98
+logo: /assets/images/meshery-logo.svg
+image: /assets/images/helm-dark.svg
+extensionInfo: |
+  Helm Plugin for Meshery Snapshots allow users to import packaged Helm charts as Meshery Designs and then Meshery Snapshots generate an image as a visual snapshot of Meshery designs.
+extensionCaveats: |
+  - A native helm CLI plugin for conveniently creating a visual snapshot of a packaged and unpackaged Helm chart
+  - Support the tradeoff between synchronous and asynchronous receipt of the snapshot
+  - Get snapshots of your infrastructure directly in your PRs
```
Comment (fitzergerald) [inline collections/_extensions/helm-plugin-for-meshery-snapshot.md:19]: There are better descriptors that can be found in the helm-kanvas-snapshot repo...

Comment (jameshorton2337) [reply collections/_extensions/helm-plugin-for-meshery-snapshot.md:19]: @Aviral0702 are you incorporating this feedback?

File: collections/_extensions/helm-plugin-for-meshery-snapshot.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+layout: item
+name: Helm Plugin for Kanvas Snapshot 
+kind: Plugin
+userName: Layer5, Inc.
```
Comment (pontusringblom) [inline collections/_extensions/helm-plugin-for-meshery-snapshot.md:5]:
```suggestion
userName: Meshery Authors
```

File: collections/_extensions/helm-plugin-for-meshery-snapshot.md (hunk 3 above)
Comment (leecalcote) [reply collections/_extensions/helm-plugin-for-meshery-snapshot.md:19]: I'm not having much luck in my communication with @Aviral0702 either. I propose that one of you move forward with changes as needed.

**Pull request closed without merging**